todo_app/tests/test_view_model.py

```diff
@@ -0,0 +1,47 @@
+from todo_app.view_models.ViewModel import ViewModel
+
+
+def test_done_items():
+  # Arrange
+  view_model = ViewModel([{"title": "Walk the dog", "status": "To Do", "id": "139fhiwg"}, 
```

Comment:
This makes me a little suspicious, isn't the ViewModel receiving a list of `Item` objects, not a list of dictionaries?

I think this _shouldn't_ work; it passes because the code isn't actually using the properties, so the behaviour is consistent between the ViewModel properties and the tests, but doesn't actually match the arguments the ViewModel receives in production!